### ISSUE TITLE
ES6 Arrow functions

### ISSUE THREAD

**Issue by Paulsky** [flot/flot#1643]
Hello,

My current project is stuck at ES5. I'm still using gulp 3.9 for the minification and 'uglification' of my JS code to generate the production files.

Because of this, I can't minify/uglify jquery.flot.js. On line **976** there is a function called '**addEventHandler**' which uses ES6 arrow functions.

Is it possible to change (back) the ES6 arrow function to ES5 anonymous functions...?

Like this:

`eventList.sort(function(a, b) { return b.priority - a.priority });`

and

```
eventList.forEach(function(eventData) {
     eventData.eventHolder.unbind(eventData.event, eventData.handler);
     eventData.eventHolder.bind(eventData.event, eventData.handler);
});
```

Instead of this:

`eventList.sort((a, b) => b.priority - a.priority );`

and

```
eventList.forEach( eventData => {
     eventData.eventHolder.unbind(eventData.event, eventData.handler);
     eventData.eventHolder.bind(eventData.event, eventData.handler);
});
```

This way, I'm able to uglify/minify the code.

Thank you in advance!

**Comment by andrewdove1** [comment]
In the npm package there is a dist/es5 folder containing the code you need

**Comment by Paulsky** [comment]
@andrewdove1 Ah, alright. I used the Bower package. Which doesn't contain the es5 folder I believe. Thank you.

**Comment by Paulsky** [comment]
I understand that the NPM package and the use of the distribution file in the ES5 folder is the current way to go. 

But unfortunately, this is not really a solution for my project. I'm using the Bower jQuery package. So I should move jQuery to NPM also (because of dependencies and deferred loading). Which will create even more breaking changes for my project. I don't really understand why the ES6 arrow functions are used. They are only used twice in the whole project (as far I found). Anyway, I created a workaround for now. Thank you!
